proto/push/service.proto

```diff
@@ -95,9 +97,139 @@ message GetPushResponse {
+            value: '{ "code": 3, "message": "invalid arguments error", "details": [] }'
+          }
+        }
+      }
+      responses: {
+        key: "401"
+        value: {
+          description: "Request could not be authenticated (authentication required)."
+          schema: { json_schema: { ref: ".google.rpc.Status" } }
+          examples: {
+            key: "application/json"
+            value: '{ "code": 16, "message": "not authenticated", "details": [] }'
+          }
+        }
+      }
+    };
+  }
+  rpc CreatePush(CreatePushRequest) returns (CreatePushResponse) {
+    option (google.api.http) = {
+      post: "/v1/push"
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Create",
+      description: "Create a push."
+      tags: "push"
+      operation_id: "web.v1.push.create"
+      responses: {
+        key: "400"
+        value: {
+          description: "Returned for bad requests that may have failed validation."
+          schema: { json_schema: { ref: ".google.rpc.Status" } }
+          examples: {
+            key: "application/json"
+            value: '{ "code": 3, "message": "invalid arguments error", "details": [] }'
+          }
+        }
+      }
+      responses: {
+        key: "401"
+        value: {
+          description: "Request could not be authenticated (authentication required)."
+          schema: { json_schema: { ref: ".google.rpc.Status" } }
+          examples: {
+            key: "application/json"
+            value: '{ "code": 16, "message": "not authenticated", "details": [] }'
+          }
+        }
+      }
+    };
+  }
+  rpc UpdatePush(UpdatePushRequest) returns (UpdatePushResponse) {
+    option (google.api.http) = {
+      patch: "/v1/push/{environment_id}/{id}"
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Update",
+      description: "Update a push."
+      tags: "push"
+      operation_id: "web.v1.push.update"
+      responses: {
+        key: "400"
+        value: {
+          description: "Returned for bad requests that may have failed validation."
+          schema: { json_schema: { ref: ".google.rpc.Status" } }
+          examples: {
+            key: "application/json"
+            value: '{ "code": 3, "message": "invalid arguments error", "details": [] }'
+          }
+        }
+      }
+      responses: {
+        key: "401"
+        value: {
+          description: "Request could not be authenticated (authentication required)."
+          schema: { json_schema: { ref: ".google.rpc.Status" } }
+          examples: {
+            key: "application/json"
+            value: '{ "code": 16, "message": "not authenticated", "details": [] }'
+          }
+        }
+      }
+    };
+  }
+  rpc GetPush(GetPushRequest) returns (GetPushResponse) {
+    option (google.api.http) = {
+      get: "/v1/push/{environment_id}/{id}"
```

Comment:
I know I asked you to do this, but I found out that we don't need it.
When using GET as a method, it will automatically read all the fields described in the request proto message.

```suggestion
      get: "/v1/push"
```

Don't forget to make the required fields. So the documentation is created correctly.

Reference:
https://github.com/bucketeer-io/bucketeer/pull/1387#discussion_r1887835166

Reply:
I have made it required fields instead of path